custom_components/dbuezas_eq3btsmart/climate.py

```diff
@@ -176,11 +182,6 @@ def available(self) -> bool:
         return True
         return self._thermostat.mode >= 0
 
-    @property
```

Comment:
Why did you remove this? Everything is now called Unnamed device
<img width="643" alt="image" src="https://user-images.githubusercontent.com/777196/202893263-b17b91f6-e893-4a43-9fc0-36f3e98b2bb6.png">

Reply:
I followed the documentation (https://developers.home-assistant.io/docs/core/entity#has_entity_name-true-mandatory-for-new-integrations) but I see the problem. I must have missed that because it works on the dashboard for some reason.

Reply:
Ok, I see that adding it makes the name appear twice in different places.
Maybe the name of the entity should be just "Thermostat", then HA would prefix it with the device name and one gets "Living Room Thermostat"

Reply:
Ah no I see my bug. I set the device name to self.name which is of course none now. It needs to be self._thermostat.name.

Reply:
That makes sense.
I already merged it so I'll do the change you mention